frontend/src/components/Movies/Movies.component.vue

```diff
@@ -229,31 +236,30 @@ export default {
       });
     },
     editButton(movieId) {
+      MoviesController.retrieve(movieId).then((response) => {
+        this.editingMovie = response.data;
```

Comment:
`response.data` je json, `editingMovie` je model, pozovi `this.editingMovie = new Movie(response.data)`